brion/blueConfig.cpp

```diff
@@ -65,6 +68,12 @@ inline std::string lexical_cast( const brion::BlueConfigSection& b )
 }
 }
 
+namespace
+{
+    const char* const SPIKE_FILE = "/out.dat";
```

Comment:
shall we group those semantics in a common place and document them once and for all so that everybody knows that they exist? Same for the strings used below like "CircuitTarget" etc.